src/style.css

```
@@ -0,0 +1,89 @@
+* {
+  margin: 0;
+  box-sizing: border-box;
+}
+
+.stars-block {
+  margin-top: 2px;
+}
+
+.stars__row {
+  display: flex;
+  align-items: center;
+  text-align: center;
+  margin-left: 2px;
+}
+
+.stars__star:not(:last-child) {
+  margin-right: 4px;
+}
+
+.stars--0 .stars__star {
+  background-image: url(../src/images/star.svg);
+  width: 16px;
+  height: 16px;
+  background-repeat: no-repeat;
+}
+
+.stars--1 .stars__star:nth-child(n + 1) {
+  background-image: url(../src/images/star.svg);
+  width: 16px;
+  height: 16px;
+  background-repeat: no-repeat;
+}
+
+.stars--1 .stars__star:nth-child(1) {
+  background-image: url(../src/images/star-active.svg);
+  width: 16px;
+  height: 16px;
+  background-repeat: no-repeat;
+}
+
+.stars--2 .stars__star:nth-child(n + 2) {
+  background-image: url(../src/images/star.svg);
+  width: 16px;
+  height: 16px;
+  background-repeat: no-repeat;
+}
+
+.stars--2 .stars__star:nth-child(-n + 2) {
+  background-image: url(../src/images/star-active.svg);
+  width: 16px;
+  height: 16px;
+  background-repeat: no-repeat;
+}
+
+.stars--3 .stars__star:nth-child(n + 3) {
+  background-image: url(../src/images/star.svg);
+  width: 16px;
+  height: 16px;
+  background-repeat: no-repeat;
+}
+
+.stars--3 .stars__star:nth-child(-n + 3) {
+  background-image: url(../src/images/star-active.svg);
+  width: 16px;
+  height: 16px;
+  background-repeat: no-repeat;
+}
+
+.stars--4 .stars__star:nth-child(n + 4) {
+  background-image: url(../src/images/star.svg);
+  width: 16px;
+  height: 16px;
+  background-repeat: no-repeat;
+}
+
+.stars--4 .stars__star:nth-child(-n + 4) {
+  background-image: url(../src/images/star-active.svg);
+  width: 16px;
+  height: 16px;
+  background-repeat: no-repeat;
+}
+
+.stars--5 .stars__star:nth-child(-n + 5) {
```

combine all modifiers like this
```suggestion
.stars--4 .stars__star:nth-child(n + 4),
.stars--5 .stars__star:nth-child(-n + 5) {
  background-image: url(../src/images/star-active.svg);
}
```